src/esp/core/ManagedContainer.h

```diff
@@ -14,14 +14,37 @@
 
 namespace esp {
 namespace core {
+
+/**
+ * @brief This enum describes how objects held in the ManagedConatainers are
```

Comment:
```suggestion
 * @brief This enum describes how objects held in the @ref ManagedConatainer are
```

(personally I'm always making sure that the enum links to the class that uses it so the relation is clear)